### PR TITLE
[bitnami/tomcat] Add prometheusRules support

### PR DESCRIPTION
**Pull request by bbaudelet**
**Description of the change**

Add Prometheus Rules support managed by prometheus operator.
Regenerate Readme.

**Benefits**

Prometheus rule can be deployed by chart without used extraDeploy or another deployment

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/tomcat/templates/prometheusrule.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+{{- with .Values.metrics.prometheusRule.namespace }}
+  namespace: {{ . }}
+{{- end }}
+  labels:
+  {{- include "common.labels.standard" . | nindent 4 }}
+  {{- if .Values.commonLabels }}
+  {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- end }}
+  {{- with .Values.metrics.prometheusRule.additionalLabels }}
+  {{- toYaml . | nindent 4 }}
```
Comment (Mauraza) [inline bitnami/tomcat/templates/prometheusrule.yaml:15]: Could you change this line to:
```suggestion
  {{- include "common.tplvalues.render" (dict "value" .Values.metrics.prometheusRule.additionalLabels "context" $) | nindent 4 }}
```

File: bitnami/tomcat/templates/prometheusrule.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+{{- with .Values.metrics.prometheusRule.rules }}
+  groups:
+    - name: {{ template "common.names.name" $ }}
+      rules: {{ tpl (toYaml .) $ | nindent 8 }}
```
Comment (Mauraza) [inline bitnami/tomcat/templates/prometheusrule.yaml:24]:
```suggestion
      rules: {{- include "common.tplvalues.render" ( dict "value" . "context" $ ) | nindent 8 }}
```

**Review by Mauraza — COMMENTED** [review]
Hi @bbaudelet,

Thanks for the contribution!! Could you check the comments?

File: bitnami/tomcat/Chart.yaml [hunk 3]
```diff
@@ -26,4 +26,4 @@ name: tomcat
 sources:
   - https://github.com/bitnami/bitnami-docker-tomcat
   - http://tomcat.apache.org
-version: 9.6.3
+version: 9.7.3
```
Comment (carrodher) [inline bitnami/tomcat/Chart.yaml:29]:
```suggestion
version: 9.7.0
```

File: bitnami/tomcat/templates/prometheusrule.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{{- if and .Values.metrics.enabled .Values.metrics.prometheusRule.enabled }}
```
Comment (Mauraza) [inline bitnami/tomcat/templates/prometheusrule.yaml:1]: The value `metrics.enabled` doesn't exist

**Review by Mauraza — COMMENTED** [review]
Hi @bbaudelet,

Could you check the small comment?

**Review by Mauraza — APPROVED** [review]
Hi @bbaudelet,

Thanks again for the contribution!! LGTM

**Pull request merged**